static/app/views/starfish/views/spanSummaryPage/index.tsx

```diff
@@ -83,49 +66,21 @@ function SpanSummaryPage({params, location}: Props) {
     queryFilter,
     [
       SpanMetricsFields.SPAN_OP,
-      SpanMetricsFields.SPAN_DESCRIPTION,
-      SpanMetricsFields.SPAN_ACTION,
-      SpanMetricsFields.SPAN_DOMAIN,
-      'count()',
-      'spm()',
-      `sum(${SpanMetricsFields.SPAN_SELF_TIME})`,
-      `avg(${SpanMetricsFields.SPAN_SELF_TIME})`,
-      'time_spent_percentage()',
-      'http_error_count()',
+      SpanMetricsFields.SPAN_GROUP,
+      `${StarfishFunctions.SPS}()`,
     ],
     'api.starfish.span-summary-page-metrics'
   );
 
   const span = {
-    ...spanMetrics,
+    [SpanMetricsFields.SPAN_OP]: spanMetrics[SpanMetricsFields.SPAN_OP],
     [SpanMetricsFields.SPAN_GROUP]: groupId,
-  } as {
-    [SpanMetricsFields.SPAN_OP]: string;
-    [SpanMetricsFields.SPAN_DESCRIPTION]: string;
-    [SpanMetricsFields.SPAN_ACTION]: string;
-    [SpanMetricsFields.SPAN_DOMAIN]: string;
-    [SpanMetricsFields.SPAN_GROUP]: string;
-  };
-
-  const {isLoading: areSpanMetricsSeriesLoading, data: spanMetricsSeriesData} =
-    useSpanMetricsSeries(
-      groupId,
-      queryFilter,
-      [`avg(${SpanMetricsFields.SPAN_SELF_TIME})`, 'spm()', 'http_error_count()'],
-      'api.starfish.span-summary-page-metrics-chart'
-    );
-
-  useSynchronizeCharts([!areSpanMetricsSeriesLoading]);
-
-  const spanMetricsThroughputSeries = {
-    seriesName: span?.[SpanMetricsFields.SPAN_OP]?.startsWith('db')
-      ? 'Queries'
-      : 'Requests',
-    data: spanMetricsSeriesData?.['spm()'].data,
   };
 
-  const title = getDescriptionLabel(span[SpanMetricsFields.SPAN_OP], true);
-  const spanDescriptionCardTitle = getDescriptionLabel(span[SpanMetricsFields.SPAN_OP]);
+  const title = [
+    getSpanOperationDescription(span[SpanMetricsFields.SPAN_OP]),
+    t('Summary'),
+  ].join(' ');
```

Comment:
l: Why the array over a templated string?

Reply:
Saw this pattern elsewhere in Performance and used it without thinking too deeply, TBH!